server/application.py

```diff
@@ -34,6 +34,9 @@
         "SERVER_TYPE",
         "COLLECT_STATISTICS",
         "USER_SELF_REGISTRATION",
+        "GLOBAL_ADMIN",
```

Comment:
do you need all of them? (I'd say READ is enough)

Reply:
I should be also GLOBAL_READ=False and GLOBAL_WRITE=True ,  no?

Reply:
ah yes, you are right